packages/strapi-plugin-graphql/package.json

```diff
@@ -13,14 +13,13 @@
   "dependencies": {
     "@apollo/federation": "^0.15.0",
     "apollo-server-koa": "2.16.0",
-    "dataloader": "^1.4.0",
+    "dataloader": "^2.0.0",
     "glob": "^7.1.4",
-    "graphql": "^14.3.0",
+    "graphql": "^15.3.0",
     "graphql-depth-limit": "^1.1.0",
     "graphql-iso-date": "^3.6.1",
-    "graphql-playground-middleware-koa": "^1.6.0",
-    "graphql-tools": "4.0.6",
-    "graphql-type-json": "0.3.1",
+    "graphql-playground-middleware-koa": "^1.6.17",
+    "graphql-type-json": "0.3.2",
```

Comment:
Isn't graphql-tools a peer dependencies of one of the other packages ?

Reply:
`graphql-tools` is a package that was used to create the GraphQL schema when the GraphQL plugin was first introduced into Strapi.
https://github.com/strapi/strapi/blob/4541b6f30bae74eff240fe8829f54f12d21b6d5c/packages/strapi-plugin-graphql/middlewares/graphql/index.js#L7-L37
The package is now embedded in apollo-server-koa. (https://github.com/apollographql/apollo-server/blob/main/package.json#L108)

The code below shows you calling a function to make a schema from `apollo-server-koa`.
https://github.com/strapi/strapi/blob/35aafde2288e6858a9b1757cf2555a8561421140/packages/strapi-plugin-graphql/services/schema-generator.js#L9

Thus, `graphql-tools` can now be safely removed from `package.json`.